### PR TITLE
[Task]add sdk apache dubbo integrated test (#3925)

### PR DESCRIPTION
**Pull request by mahaitao617**
Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-integrated-test/shenyu-integrated-test-sdk-apache-dubbo/pom.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>${docker-maven-plugin.version}</version>
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>apache/shenyu-integrated-test-apache-dubbo</name>
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-sdk-apache-dubbo/pom.xml:126]:
```suggestion
                                    <name>apache/shenyu-integrated-sdk-test-apache-dubbo</name>
```

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: shenyu-integrated-test/shenyu-integrated-test-sdk-apache-dubbo/pom.xml (hunk 1 above)
Comment (mahaitao617) [reply shenyu-integrated-test/shenyu-integrated-test-sdk-apache-dubbo/pom.xml]: thanks for your time

File: shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java [hunk 2]
```diff
@@ -225,6 +224,7 @@ public <S> S getFromGateway(final String path, final Type type) throws IOExcepti
      * @return response from gateway
      * @throws IOException IO exception
      */
+
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java:227]: Please keep the code style~

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java (hunk 2 above)
Comment (JooKS-me) [reply shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java:227]: Do not add blank lines here

**Review by JooKS-me — APPROVED** [review]
Good job!

**Pull request merged**